set.mm

```diff
@@ -419561,6 +419601,26 @@ the sequence p(0) p(1) ... p(n-1) of the vertices in a closed walk p(0)
       BUCUHCDABOPUFUJDCUGUDCLUEUIBUDCJRSTUA $.
   $}
 
+  ${
+    $d g n w $.
+    $( Conditions for ` ClWWalksN ` not being defined.  (Contributed by
```

Comment:
"Sufficient condition for..."
?

Reply:
Only one of the disjuncts in the antecedent would already be sufficient. Therefore I would not add this here. Mario sometimes comments such theorems by "... equals the empty set when not meaningful" (see ~riotaund or ~oi0).